static/js/common/init.js

```diff
@@ -24,24 +24,63 @@ MEDUSA.common.init = function() {
         return offset;
     }
 
+    let initHorizontalScroll = function() {
```

Comment:
What exactly is this function doing? I have a feeling there's a way better way todo this.